### ISSUE TITLE
Failed to place: IP is overused

### ISSUE THREAD

**Issue by TexasTexasTexas** [Topinambur223606/PixelPlanetTools#41]
Does it on --zeroCooldown, and regular placing mode. Due to this issue, images which should have been done hours ago have 10+ hours left on a small image. Any help is apreciated. Cause of this problem is unknown, just happens randomly. Note: After problem starts, on cooldown mode the bot starts to place pixels twice a second. Thanks for reading

Texas
![image](https://user-images.githubusercontent.com/117549957/200148454-cbc0e02c-205e-4bb1-84bc-ec3c24db52b5.png)

(Note: issue is in the title)

**Comment by Topinambur223606** [comment]
If I understand correctly, we are talking about a self-host.
Are you doing this as admin?
For zero cooldown to work, you need to log in using admin account and specify created session afterwards.

**Comment by TexasTexasTexas** [comment]
Yes, we are on a self host. My friend who is not an admin on the server has zero problems with --zeroCooldown, while the rest of us do. None of us have admin accounts. Is there a way we can set the ip overused cooldown to 5 seconds for example?

**Comment by TexasTexasTexas** [comment]
Also, if i do the command without --zeroCooldown, after 5 minutes it gives an ip overused prompt then places 2 pixels a second.

**Comment by Topinambur223606** [comment]
I believe something is incorrect with server configuration or within its code, there is not much I can do about it in the bot.
Gonna check if auth works, but not sure if it can be the cause anyways.
